src/app/pages/test-report/components/reverse-left/reverse-left.actions.ts

```diff
@@ -0,0 +1,9 @@
+import { createAction } from '@ngrx/store';
+
+export const ReverseLeftPopoverOpened = createAction(
+  '[Reverse left] Reverse Left Popover Opened',
+);
+
+export const ReverseLeftPopoverClosed = createAction(
+  '[Reverse left] Reverse Left Popover Closed',
+);
```

Comment:
there is no union function in this file, not sure if we are trying to be consistent with it or not

Reply:
only adding if they exist in previous app for now. As discussed, we can probably remove them all once all reducers are refactored